### ISSUE TITLE
Search results remain highlighted after closing unfinished search

### ISSUE THREAD

**Issue by andy-ms** [Microsoft/vscode#48073]
Issue Type: <b>Bug</b>

* Open [`DefinitelyTyped`](https://github.com/DefinitelyTyped/DefinitelyTyped). (This is hard to reproduce in a small project because it relies on search taking some time.)
* Open `types/abs/index.d.ts`.
* Press `ctrl-shift-f` to open find-all
* Type `ex` and search, and quickly press `ctrl-b` to close the find window before it finishes searching.
* `ex` remains highlighted.

![ex](https://user-images.githubusercontent.com/19274678/38893735-0ac98044-4240-11e8-9094-a6083537da13.jpg)
0.238Z)

VS Code version: Code - Insiders 1.23.0-insider (ab56a0952b9c2ff7add6404f80ebf9398cf8ae51, 2018-04-17T05:12:5
OS version: Linux x64 4.13.0-37-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz (8 x 2600)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: unavailable_software<br>video_encode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 2, 2|
|Memory (System)|7.63GB (1.90GB free)|
|Process Argv|/usr/share/code-insiders/code-insiders .|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
typescript-javascript-grammar|ms-|0.0.39

</details>

**Comment by andy-ms** [comment]
Reproduced this again with vscode-insiders that I updated this morning.

Version: 1.28.0-insider
Commit: 8330a695712c43f861ce5c47533d82637d3cb374
Date: 2018-09-13T11:25:01.441Z

![image](https://user-images.githubusercontent.com/19274678/45502825-d76c0680-b739-11e8-84ed-4eaa625d25c0.png)

I can't think of much to elaborate on from my initial description. Note that this needs to be done *fast*!  After typing `ex` in the find-all bar, *immediately* hit enter then `ctrl-b`. It looks like the highlight on `ex` actually goes away for a second before coming back.
